packages/manager/apps/cloud/package.json

```diff
@@ -113,7 +113,7 @@
     "ng-slide-down": "TheRusskiy/ng-slide-down#^1.0.0",
     "office-ui-fabric-core": "^11.0.0",
     "ovh-angular-list-view": "ovh-ux/ovh-angular-list-view#^0.1.5",
-    "ovh-api-services": "^9.50.0",
+    "ovh-api-services": "^10.0.0",
```

Comment:
`ovh-api-services` is already bumped to `"ovh-api-services": "^10.0.0",` in `develop` (see https://github.com/ovh/manager/blob/develop/packages/manager/apps/cloud/package.json#L117 )

Reply:
it shouldn't break anything, you can also rebase your branch on develop and the diff should not appear anymore